wpilibj/src/main/java/edu/wpi/first/wpilibj/NidecBrushless.java

```diff
@@ -0,0 +1,221 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2008-2017 FIRST. All Rights Reserved.                        */
```

Comment:
Copyright year is wrong. By the way, you can just leave the copyright header out for new files and wpiformat will generate and insert one for you.